src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -350,6 +351,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Function
 
+        Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, options As GetOperationOptions, cancellationToken As CancellationToken) As IOperation
+            Dim model As MemberSemanticModel = Me.GetMemberSemanticModel(node)
+
+            If model IsNot Nothing Then
```

Comment:
I'm inclined to leave this looking like the rest of the file, and I'm not keen on introducing stylistic cleanup to surrounding code in this PR.